<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>moneySelect</class>
 <widget class="QWidget" name="moneySelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="twentyEuro">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>20 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fortyEuro">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>40 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fiftyEuro">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>50 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hundredEuro">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>100 €</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chooseEuro">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Muu summa</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
